integration-cli/docker_cli_run_test.go

```diff
@@ -3945,11 +3945,12 @@ func (s *DockerSuite) TestRunAttachFailedNoLeak(c *testing.T) {
 	assert.Assert(c, err != nil, "Command should have failed but succeeded with: %s\nContainer 'test' [%+v]: %s\nContainer 'fail' [%+v]: %s", out, err1, out1, err2, out2)
 	// check for windows error as well
 	// TODO Windows Post TP5. Fix the error message string
-	assert.Assert(c, strings.Contains(out, "port is already allocated") ||
-		strings.Contains(out, "were not connected because a duplicate name exists") ||
-		strings.Contains(out, "The specified port already exists") ||
-		strings.Contains(out, "HNS failed with error : Failed to create endpoint") ||
-		strings.Contains(out, "HNS failed with error : The object already exists"), fmt.Sprintf("Output: %s", out))
+	outLowerCase := strings.ToLower(out)
+	assert.Assert(c, strings.Contains(outLowerCase, "port is already allocated") ||
+		strings.Contains(outLowerCase, "were not connected because a duplicate name exists") ||
+		strings.Contains(outLowerCase, "the specified port already exists") ||
+		strings.Contains(outLowerCase, "hns failed with error : failed to create endpoint") ||
+		strings.Contains(outLowerCase, "hns failed with error : the object already exists"), fmt.Sprintf("Output: %s", out))
```

Comment:
Looks like this actually was hiding a regression;

- https://github.com/moby/moby/pull/49202

Introduced in (upstream);

- https://github.com/microsoft/hcsshim/pull/573